snaplet-sqlite-simple.cabal

```diff
@@ -1,5 +1,5 @@
 name:           snaplet-sqlite-simple
-version:        0.5
+version:        0.5.0.0
```

Comment:
might as well bump to 0.6.0.0, this version probably drops ghc 7.6 support or at least increases dep requirement so much that we shouldn't call this version 0.5.0.0 (which would be equal to 0.5.*, ie. same as the previous release)